code/mapping/src/mapping_data_integration.py

```
@@ -13,10 +13,11 @@
 from mapping_common.transform import Transform2D, Vector2
 from mapping_common.shape import Circle, Rectangle
 from mapping_common.map import Map
-from mapping.msg import Map as MapMsg
+from mapping.msg import Map as MapMsg, PointcloudClusterArray, ClusteredLidarPoints
```

Ruff (0.8.2)

`mapping.msg.PointcloudClusterArray` imported but unused

Remove unused import: `mapping.msg.PointcloudClusterArray`

(F401)